### ISSUE TITLE
Pkgdown is failing

### ISSUE THREAD

**Issue by elinw** [ropensci/skimr#700]
Hi @michaelquinn32 

The pkgdown is failing

```
Error in `.f()`:
! Failed to parse Rd in focus.Rd
ℹ no applicable method for 'replay_html' applied to an object of class "c('one_skim_df', 'tbl_df', 'tbl', 'data.frame')"
Caused by error in `UseMethod()`:
! no applicable method for 'replay_html' applied to an object of class "c('one_skim_df', 'tbl_df', 'tbl', 'data.frame')"
```

I don't think this matters for CRAN but do you have any clue?  Or @maelle  do you know?

**Comment by maelle** [comment]
:wave: @elinw (and @michaelquinn32)!

Interesting :sweat_smile: I see https://github.com/r-lib/pkgdown/issues/677

Does the `one_skim_df` have a print method? I'll try to write a reprex for downlit. :thinking:

**Comment by maelle** [comment]
A reprex would be this code

```r
code <- "# Compare
iris %>%
  skim() %>%
  dplyr::select(n_missing)

iris %>%
  skim() %>%
  focus(n_missing)

# This is equivalent to
iris %>%
  skim() %>%
  dplyr::select(skim_variable, skim_type, n_missing)"

downlit::evaluate_and_highlight(code)
```

but not if run with `reprex::reprex()` which might be due to the knitr stuff reprex itself does.

**Comment by maelle** [comment]
Maybe https://github.com/r-lib/pkgdown/issues/677#issuecomment-955025248 is a good thing to try (i.e. returning the object invisibly if not already the case)

**Comment by elinw** [comment]
Thank you for all of this. I'm going to try the invisibility approach. We do have custom objects and I guess something must have changed.  We only build the site on a merge to main, so it has been a while.

**Comment by elinw** [comment]
@michaelquinn32   It looks to me like we need an explicit `print.one_skim_df()`  function.  I tried a bit but hit a dead end.

**Comment by elinw** [comment]
@michaelquinn32  I'm going to create a new issue, I thin we need to do a quick new release that I'll explain there.

**Comment by michaelquinn32** [comment]
Fixed with the release of 2.1.4
